.gitignore

```diff
@@ -10,7 +10,8 @@ node_modules/
 /storybook-static/
 /packages/**/dist/
 
-/.env.*
+/.env.local
+/.env.*.local
```

Comment:
Not technically needed, but more accurate: ignores `.env.test.local` but not `.env.test`, for instance.